frontend/src/scenes/pipeline/destinations/destinationsFiltersLogic.tsx

```diff
@@ -59,26 +60,31 @@ export const destinationsFiltersLogic = kea<destinationsFiltersLogicType>([
         },
 
         openFeedbackDialog: async (_, breakpoint) => {
+            const isTransformation = values.types.includes('transformation')
+            const itemType = isTransformation ? 'transformation' : 'destination'
+
             await breakpoint(100)
             LemonDialog.openForm({
-                title: 'What destination would you like to see?',
+                title: `What ${itemType} would you like to see?`,
                 initialValues: { destination_name: values.filters.search },
                 errors: {
                     destination_name: (x) => (!x ? 'Required' : undefined),
                 },
                 description: undefined,
                 content: (
                     <div className="space-y-2">
-                        <LemonField name="destination_name" label="Destination">
-                            <LemonInput placeholder="What destination would you like to see?" autoFocus />
+                        <LemonField name="destination_name" label={isTransformation ? 'Transformation' : 'Destination'}>
+                            <LemonInput placeholder={`What ${itemType} would you like to see?`} autoFocus />
                         </LemonField>
                         <LemonField name="destination_details" label="Additional information" showOptional>
-                            <LemonTextArea placeholder="Any extra details about what you would need this destination to do or your overall goal" />
+                            <LemonTextArea
+                                placeholder={`Any extra details about what you would need this ${itemType} to do or your overall goal`}
+                            />
                         </LemonField>
                     </div>
                 ),
                 onSubmit: async (values) => {
-                    posthog.capture('cdp destination feedback', { ...values })
+                    posthog.capture(`cdp ${itemType} feedback`, { ...values })
```

Comment:
Just an FYI to update [this](https://us.posthog.com/project/2/pipeline/destinations/hog-0190e3d7-0a2a-0000-3bca-872b39e610b0/configuration?search=cdp) after release